hamlpy/test/templates/filters.html

```diff
@@ -24,3 +24,4 @@
 3
 4
 5
+
```

Comment:
This is one of a few cases where we don't match the original output exactly because of an additional new line. IMO this new line is correct (it's coming from the python `print`) and shouldn't break anyone's stuff.